datafusion/core/src/datasource/listing/table.rs

```diff
@@ -450,7 +451,7 @@ impl ListingOptions {
 }
 
 /// Reads data from one or more files via an
-/// [`ObjectStore`](object_store::ObjectStore). For example, from
+/// [`ObjectStore`]. For example, from
```

Comment:
doc check fail:
```
 Documenting test-utils v0.1.0 (/__w/arrow-datafusion/arrow-datafusion/test-utils)
error: redundant explicit link target
   --> datafusion/core/src/datasource/listing/table.rs:454:21
    |
454 | /// [`ObjectStore`](object_store::ObjectStore). For example, from
    |      -------------  ^^^^^^^^^^^^^^^^^^^^^^^^^ explicit target is redundant
    |      |
    |      because label contains path that resolves to same destination
    |
    = note: when a link's destination is not specified,
            the label is used to resolve intra-doc links
    = note: `-D rustdoc::redundant-explicit-links` implied by `-D warnings`
    = help: to override `-D warnings` add `#[allow(rustdoc::redundant_explicit_links)]`
help: remove explicit link target
    |
454 | /// [`ObjectStore`]. For example, from
    |     ~~~~~~~~~~~~~~~

 Documenting datafusion-physical-plan v36.0.0 (/__w/arrow-datafusion/arrow-datafusion/datafusion/physical-plan)
error: could not document `datafusion`
```